dds_web/utils.py

```diff
@@ -319,6 +320,28 @@ def page_query(q):
             break
 
 
+def create_one_time_password_email(user, hotp_value):
+    msg = flask_mail.Message(
+        "DDS One-Time Authentication Code",
+        sender=flask.current_app.config.get("MAIL_SENDER", "dds@noreply.se"),
```

Comment:
I can't find `MAIL_SENDER` in `congif.py`. Either add it or set `MAIL_DEFAULT_SENDER` instead, not setting the sender here.